CMakeLists.txt

```diff
@@ -1008,6 +1008,32 @@ endif ()
 #-----------------------------------------------------------------------------
 if (EXISTS "${HDF5_SOURCE_DIR}/utils" AND IS_DIRECTORY "${HDF5_SOURCE_DIR}/utils")
   add_subdirectory (utils)
+  option (HDF5_BUILD_PARALLEL_TOOLS  "Build Parallel HDF5 Tools" OFF)
+  if (HDF5_BUILD_PARALLEL_TOOLS AND HDF5_ENABLE_PARALLEL)
+    set (CMAKE_PREFIX_PATH "$HDF_RESOURCES_DIR")
+    find_package(MFU REQUIRED)
+    if (MFU_FOUND)
+      message(STATUS "LL_PATH=${LL_PATH}")
+      set (ENV{H5DWALK_TEST_ENV} "${LL_PATH}")
```

Comment:
Where is H5DWALK_TEST_ENV used?

Reply:
It's used now in utils/tools/test/h5dwalk/CMakeTests.cmake.  It provides the library path for TEST_LIBRARY_DIRECTORY.  (It works for me, and presumably for Richard, too.)

Reply:
However if the suggested changes are made - this should be unnecessary.

Reply:
Do you mean adding -D "TEST_LIBRARY_DIRECTORY=${LL_PATH}" to CMakeTests.cmake?  When I did that, the result in build/utils/tools/test/h5dwalk/CTestTestfile.cmake was "-D" "TEST_LIBRARY_DIRECTORY=", but with -D "TEST_LIBRARY_DIRECTORY=$ENV{H5DWALK_TEST_ENV}" it is "-D" "TEST_LIBRARY_DIRECTORY=/mnt/hdf/packages/mpifileutils/install/lib64:/mnt/hdf/packages/mpifileutils/install/lib".